docs/user/managing-devices.md

```diff
@@ -505,6 +507,99 @@ LABEL appType="compose"
+| EnvVars | (Optional) A list of key/value-pairs to set as environment variables for the command. |
+| WorkDir | (Optional) The directory the command will be run from. |
+| Timeout | (Optional) The maximum duration allowed for the action to complete. The duration must be be specified as a single positive integer followed by a time unit. Supported time units are `s` for seconds, `m` for minutes, and `h` for hours. |
+| If | (Optional) A list of conditions that must be true for the action to be run (see below). |
+
+By default, actions are performed every time the hook is triggered. However, for the `afterUpdating` hook you can use the `If` parameter to add conditions that must be true for an action to be performed, otherwise the action will be skipped.
+
+In particular, to only run an action if a given file or directory has changed during the update, you can define a "path condition" that takes the following parameters:
+
+| Parameter | Description |
+| --------- | ----------- |
+| Path | An absolute path to a file or directory that must have changed during the update as condition for the action to be performed. Paths must be specified using forward slashes (`/`) and if the path is to a directory it must terminate with a forward slash `/`.<br/></br>If you specify a path to a file, the file must have changed to satisfy the condition.</br>If you specify a path to a directory, a file in that directory or any of its subdirectories must have changed to satisfy the condition.|
+| On | A list of operations (`created`, `updated`, `removed`) to further limit the kind of changes to the specified path as condition for the action to be performed. |
+
+If you have specified a "path condition" for an action in the `afterUpdating` hook, you have the following variables that you can include in arguments to your command and that will be replaced with the absolute path(s) to the changed files:
+
+| Variable | Description |
+| -------- | ----------- |
+| `{{ Path }}` | The absolute path to the file or directory specified in the path condition. |
+| `{{ Files }}` | A space-separated list of absolute paths of the files that were changed (created, updated, or removed) during the update and are covered by the path condition. |
+| `{{ CreatedFiles }}` | A space-separated list of absolute paths of the files that were changed (created, updated, or removed) during the update and are covered by the path condition. |
+| `{{ UpdatedFiles }}` | A space-separated list of absolute paths of the files that were updated during the update and are covered by the path condition. |
+| `{{ RemovedFiles }}` | A space-separated list of absolute paths of the files that were removed during the update and are covered by the path condition. |
+
+The Flight Control Agent comes with a built-in set of rules defined in `/usr/lib/flightctl/hooks.d/afterupdating/00-default.yaml`:
```

Comment:
is this true? What pipelines will deliver this agent with this RPM as the base? all?

Reply:
I've added this file to the `.spec` file, so when the flightctl-agent.rpm gets installed, the file should be in that location.

Reply:
Right and every downstream build pipeline will use that as the source.

Reply:
Or I guess if they choose not to include it they can handle downstream docs. lgtm